doc/pages/architecture/090-database.md

```diff
@@ -0,0 +1,219 @@
+# Astarte Database
+
+Astarte might need to store a massive ammount of data, to accomplish this task a [Cassandra](http://cassandra.apache.org/) database is used. Cassandra offers scalability and high availability with [good performances](https://www.datastax.com/apache-cassandra-leads-nosql-benchmark).
```

Comment:
Scrape out the first sentence.

`Astarte leverages [Cassandra](http://cassandra.apache.org/) to store all of its data, including data ingested from devices (which might scale to insane amounts)`